src/clients/OtterspaceSubgraph.ts

```diff
@@ -0,0 +1,128 @@
+import fetch from 'isomorphic-fetch'
+
+import { DAO_RAFT_ID, OTTERSPACE_QUERY_ENDPOINT } from '../entities/Snapshot/constants'
+
+import { inBatches, trimLastForwardSlash } from './utils'
+
+const BADGES_QUERY = `
+query Badges($raft_id: String!, $address: Bytes!, $first: Int!, $skip: Int!) {
+  badges: badges(
+    where: {owner: $address}, 
+    filter: {spec: {raft:{id:$raft_id}}}
+    first: $first, skip: $skip,
+    orderBy: createdAt
+    orderDirection: desc
+  ) {
+    id
+    createdAt
+    status
+    statusReason
+    spec {
+      id
+      metadata {
+        name
+        description
+        expiresAt
+        image
+      }
+      raft {
+        id
+        metadata {
+          name
+          image
+        }
+      }
+    }
+  }
+}`
+
+export enum BadgeStatus {
+  BURNED = 'BURNED',
+  MINTED = 'MINTED',
```

Comment:
We mostly use camelCase with upper first as a pattern in enums.
`Minted = 'Minted'`

Reply:
Yeah, Otterspace status are allcaps but I can change the parsing